### PR TITLE
add default tags to main.tf

### PR DESCRIPTION
**Pull request by millipz**

### CODE REVIEW

File: terraform/main.tf
Comment (bhwood) [inline terraform/main.tf:1]: Looks good.

Like the idea of default tagging.

File: terraform/test_db.tf
Comment (bhwood) [inline terraform/test_db.tf:1]: nice work. Could we potentially look at something like [this](https://developer.hashicorp.com/terraform/tutorials/configuration-language/sensitive-variables) for storing the database username and password from a security point of view when we create the instance?

File: .gitignore
Comment (bhwood) [inline .gitignore:1]: all good!

**Review by bhwood — CHANGES_REQUESTED** [review]
Looking good. Would you be able to have a look at the comment on the test_db.tf and let me know your thoughts.

Thanks

**Comment by millipz** [comment]
Closed as merged with #39

**Pull request closed without merging**